gradle/impl-gradle/src/main/java/org/jboss/shrinkwrap/resolver/impl/gradle/GradleDependency.java

```diff
@@ -0,0 +1,45 @@
+package org.jboss.shrinkwrap.resolver.impl.gradle;
+
+public class GradleDependency {
```

Comment:
True this class worked when I thought we could resolve individual dependencies. I am going to remove for now.